styles/util/_organisms.scss

```diff
@@ -4,6 +4,24 @@
  * Utility classes for organism mixins
  */
 
-.card {
-  @include organisms.card;
+/**
+  * Usage:
+  * 1. Panel with title and no close button
+  * <div class="panel">
+  *  <header><h2 class="panel__title">Panel title</h2></header>
+  *  <div>This panel has no close button</div>
+  * </div>
+  *
+  * 2. Panel with title and close button
+  * <div className="panel--closeable">
+  *   <header>
+  *     <h2 class="panel__title">Panel title</h2>
+  *     <IconButton icon="cancel" title="Close" />
```

Comment:
This is slightly inaccurate. The markup should be:

```html
<header>
  <h2 class...></h2>
  <div class="panel__close">
    <IconButton... />
  </div>
</header>
```

I'll push a fixup commit